.readthedocs.yaml

```diff
@@ -12,7 +12,16 @@ build:
     python: "3.11"
   jobs:
     post_checkout:
-      - python3 .sphinx/build_requirements.py
+    - git fetch --unshallow || true
+    # Cancel building pull requests when there aren't changed in the docs directory.
+    # If there are no changes (git diff exits with 0) we force the command to return with 183.
+    # This is a special exit code on Read the Docs that will cancel the build immediately.
+    # https://docs.readthedocs.io/en/stable/build-customization.html#cancel-build-based-on-a-condition
+    - |
+      if [ "$READTHEDOCS_VERSION_TYPE" = "external" ] && git diff --quiet origin/main -- '/' '.readthedocs.yaml';
+      then
+        exit 183;
+      fi
```

Comment:
I'd suggest deleting these lines, they're not needed since we don't have a docs dir and we are okay to always rebuild the docs on prs.

Reply:
Removing this seems to have broken the RTD check. Reinstating...